app/pages/lab/visibility.cjsx

```diff
@@ -229,6 +240,18 @@ module.exports = React.createClass
             </tbody>
           </table>
         </div>}
+
+      {if @state.workflows.length > 0
+        meta = @state.workflows[0].getMeta()
+
+        <Paginator
+          className="paginator"
```

Comment:
it looks like the Paginator component gets the paginator class by default [here](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/talk/lib/paginator.cjsx#L72), so I don't think we need to add on like 248, but please confirm on your local branch.